pkg/v1/tkg/web/src/app/views/landing/aws-wizard/node-setting-step/node-setting-step.component.ts

```diff
@@ -289,21 +289,25 @@ export class NodeSettingStepComponent extends StepFormDirective implements OnIni
                 const prodFields = ['awsNodeAz2', 'awsNodeAz3', 'workerNodeInstanceType2', 'workerNodeInstanceType3', 'prodInstanceType'];
 
                 prodFields.forEach(attr => this.disarmField(attr, true));
+                if (this.nodeAzs && this.nodeAzs.length === 1) {
+                    this.formGroup.get('awsNodeAz1').setValue(this.nodeAzs[0].name);
+                }
 
                 if (this.clusterType !== 'standalone') {
-                    this.resurrectField('workerNodeInstanceType1', [Validators.required]);
+                    this.resurrectField('workerNodeInstanceType1', [Validators.required],
+                        this.azNodeTypes.awsNodeAz1.length === 1 ? this.azNodeTypes.awsNodeAz1[0] : '');
```

Comment:
Maybe we should say that if there is only one option (which we will automatically select) then there's no need for a validator?

Reply:
@swalner-vmware  we have an empty option in all dropdown menus for unselecting. If we removed it, it won't be consistent design. In addition, the empty option is useful for the optional fields. so I think the current approach is fine.